src/unix/aix-common.c

```diff
@@ -253,25 +266,50 @@ int uv_interface_addresses(uv_interface_address_t** addresses, int* count) {
     sa_addr = (struct sockaddr_dl*) &p->ifr_addr;
     memcpy(address->phys_addr, LLADDR(sa_addr), sizeof(address->phys_addr));
 
-    if (ioctl(sockfd, SIOCGIFNETMASK, p) == -1) {
-      uv__close(sockfd);
-      return UV_ENOSYS;
-    }
-
-    if (inet6)
-      address->netmask.netmask6 = *((struct sockaddr_in6*) &p->ifr_addr);
-    else
+    if (inet6) {
+      memset(&if6, 0, sizeof(if6));
+      r = uv__strscpy(if6.ifr_name, p->ifr_name, sizeof(if6.ifr_name));
+      if (r == UV_E2BIG)
+        goto cleanup;
+      else
+        r = 0;
```

Comment:
Suggestion: drop the `else` and dedent this line.

Reply:
done